src/main/java/org/alliancegenome/curation_api/controllers/crud/ontology/ObiTermCrudController.java

```diff
@@ -0,0 +1,28 @@
+package org.alliancegenome.curation_api.controllers.crud.ontology;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+
+import org.alliancegenome.curation_api.controllers.base.BaseOntologyTermController;
+import org.alliancegenome.curation_api.dao.ontology.ObiTermDAO;
+import org.alliancegenome.curation_api.interfaces.crud.ontology.ObiTermCrudInterface;
+import org.alliancegenome.curation_api.model.entities.ontology.OBITerm;
+import org.alliancegenome.curation_api.services.helpers.GenericOntologyLoadConfig;
+import org.alliancegenome.curation_api.services.ontology.ObiTermService;
+
+@RequestScoped
+public class ObiTermCrudController extends BaseOntologyTermController<ObiTermService, OBITerm, ObiTermDAO> implements ObiTermCrudInterface {
+
+	@Inject
+	ObiTermService obiTermService;
+
+	@Override
+	@PostConstruct
+	public void init() {
+		GenericOntologyLoadConfig config = new GenericOntologyLoadConfig();
+		config.setLoadOnlyIRIPrefix("OBI");
```

Comment:
@markquintontulloch can this work without setting the prefix?

Reply:
sorry, not sure what you mean

Reply:
The helper just checks that the IRI short form starts with the prefix set here:
`boolean condition 4 = currentTreeNode.getIRI().getShortForm().startsWith(config.getLoadOnlyIRIPrefix() + "_");`

Reply:
right but that is in a OR condition if condition 1 and 2 then it doesn't need 3 and 4

Reply:
ah, I see what you mean - no, it wouldn't work as there is no namespace set
`private boolean isNodeInOntology(OWLClass currentTreeNode, T currentTerm, HashSet<String> requiredNamespaces) {
		boolean condition1 = currentTerm.getNamespace() != null;
		boolean condition2 = requiredNamespaces.contains(currentTerm.getNamespace());
		boolean condition3 = config.getLoadOnlyIRIPrefix() != null;
		boolean condition4 = currentTreeNode.getIRI().getShortForm().startsWith(config.getLoadOnlyIRIPrefix() + "_");

		boolean condition5 = !config.getIgnoreEntitiesWithChebiXref();
		boolean condition6 = !hasChebiXref(currentTerm);

		return ((condition1 && condition2) || (condition3 && condition4)) && (condition5 || condition6);
	}`

Reply:
ahh ok... just checking